packages/performance/src/services/perf_logger.ts

```diff
@@ -248,3 +246,4 @@ function getApplicationInfo(firebaseApp: FirebaseApp): ApplicationInfo {
     application_process_state: 0
   };
 }
+/* eslint-enable camelcase */
```

Comment:
It turns out this entire file requires camelcase to be disabled (now that it's spelled correctly).